app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -87,6 +87,8 @@ class ResourcesFragment : MvpAppCompatFragment(), ResourcesView {
 
     private var tagsSelectorAdapter: TagsSelectorAdapter? = null
 
+    private var shuffledState = -1
```

Comment:
Looks like it should be plain `Boolean` :)

Reply:
Boolean brings conflicts when being updated with clicks on the button

Reply:
I will find a better way to use it though :)